### ISSUE TITLE
Save without water or lava

### ISSUE THREAD

**Issue by LionJim** [andrewphorn/ClassiCube-Client#314]
One of the differences between online and local maps is active blocks, locally door become normal blocks and water/lava become active. I use water for elevators, door switches, teleports, and computer displays. I have lava for miscellaneous decoration. I would like an option to save without active blocks, so my maps don't become water messes or lava survival.

**Comment by Hetal728** [comment]
ClassiCube itself supports lava and water physics, these cannot be turned off or on from what I know. Therefore, you need to use stillwater instead of water.

**Comment by LionJim** [comment]
I included 'became active' to note that I used StillWater and StillLava. They work as expected online. StillWater behaves as Active_Water locally. On my test the only differences between StillWater and Active_water locally is how many blocks it takes to activate. Put down one StillWater and it stays, put down another StillWater next to it you get flowing. The same with lava.

Should this be a bug report?

**Comment by Hetal728** [comment]
It seems so, but I will allow someone more experienced with the client to respond (fragmer, Jonty800, Andrewphorn, 123DMWM)

**Comment by 123DMWM** [comment]
This is due to block updates, it happens in premium as well.

**Comment by n00b7** [comment]
yeah because still lava loads in then a block next to it then i detects the block update and state becomes active

**Comment by andrewphorn** [comment]
This behavior is not a bug.
